CHANGES

```diff
@@ -9,19 +24,22 @@ Version 0.4.0 (released 2015-03-11)
   provided. (#50)
 - PEP8 improvements.
 
-Version 0.3.0 (released 2014-11-05):
+Version 0.3.0 (released 2014-11-05)
+-----------------------------------
 
```

Comment:
I have removed it. However, we should change theme for one that has better support for retendering side panel.